lib/rdoc/token_stream.rb

```diff
@@ -105,13 +105,15 @@ def pop_token
   # Current token stream
 
   def token_stream
-    @token_stream || []
+    @token_stream
   end
 
   ##
   # Returns a string representation of the token stream
 
   def tokens_to_s
+    return token_stream.to_s unless token_stream
+
     token_stream.compact.map { |token| token[:text] }.join ''
```

Comment:
Using `Array()` could unify this code:

```suggestion
    Array(token_stream).compact.map { |token| token[:text] }.join ''
```

Alternatively, returning `+""` might be more explicit:

```suggestion
    return +"" if token_stream.nil?

    token_stream.compact.map { |token| token[:text] }.join ''
```

However, I am not an RDoc committer, and they may have a different opinion. :smile:

Reply:
> Using `Array()` could unify this code:

Okay, I'll take it, thanks.